libs/wingsdk/src/std/array.ts

```diff
@@ -10,14 +10,16 @@ import { Code, InflightClient } from "../core";
  *
  * @typeparam T1
  */
-export class ImmutableArray {
+export class Array {
   /**
    * @internal
    */
   public static _toInflightType(): Code {
     return InflightClient.forType(__filename, this.name);
   }
 
+  private constructor() {}
```

Comment:
Is this needed?

Reply:
It's not needed for anything in the compiler, but when the constructor is private, the "Initializer" section is removed the generate docs - which is nice since we don't want to advertise to users that these classes are meant to be initialized with `new std.String` etc. Of course we can customize how we generate docs for `std` types differently / revert this if needed in the future